setup.py

```diff
@@ -50,7 +50,7 @@
     license="Apache",
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     install_requires=[
-        "torch>1.3.1,<=1.5.0",
+        "torch>1.3.1,<1.5.1",
```

Comment:
What about `<1.6.0`? PyTorch "patch" version changes shouldn't be breaking. But maybe you think it's better to have them CI-tested anyway?

Reply:
Our CI will always test against the newest allowed package, so we'd know right away if a new patch version breaks anything. So yea, I agree with you, we should make it `<1.6.0`